README.rst

```diff
@@ -235,6 +236,44 @@ Delete
     print(response)
     # <class pybutton.Response >
 
+Customers
+~~~~~~
+
+Create
+''''''
+
+.. code:: python
+
+    import hashlib
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    hashed_email = hashlib.sha256('user@example.com'.lower().strip()).hexdigest()
+
+    response = client.customers.create({
```

Comment:
@griffinmyers it's listed in the [curl request](https://developer.usebutton.com/api?shell#post-v1-customer) but not the Ruby client. Not sure who added that. Is this a mistake?